tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -446,14 +446,19 @@ stages:
         python tools/ci_build/github/apple/build_apple_framework.py \
           --build_dir "$(Build.BinariesDirectory)/ios_framework" \
           --build_dynamic_framework \
-          tools/ci_build/github/apple/default_full_apple_framework_build_settings.json
+          tools/ci_build/github/apple/default_mobile_ios_framework_build_settings.json
       displayName: "Build iOS dynamic framework"
 
+    - script: |
+        which realpath || brew install coreutils
```

Comment:
is this still necessary? if the issue only shows up for running the tests on macos, we can fix it properly later.